src/templates/docs/font-family/index.html

```diff
@@ -47,26 +47,37 @@ <h4 class="f4 mv0 fw6 dib mr4"><%= name %></h4>
       <div class="ph3 ph5-ns pt4 pb5">
         <h2 class="f3 bold">Examples</h2>
         <h3 class="f5 book sans-serif pt4 caps">Default System Sans Serif</h3>
+        <code class="db f6">.system-sans-serif</code>
```

Comment:
Added the `system-sans-serif` example, users may miss that its a possibility if they don't check out the source code snippet at the bottom of the page